src/providers/toast.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { ToastController } from 'ionic-angular';
+
+@Injectable()
+export class ToastService {
+    private faveToast; 
+    constructor(private toast: ToastController) { } 
+    
+    favoriteToast(routeOrStop: string, isFave: boolean): void {
```

Comment:
What if we made this function private and had two public helper functions - `favoriteRouteToast` and `favoriteStopToast` which call this one with the proper parameters?  idk, just something about this method seems off to me. Interested in someone like @werebus or @dfaulken's opinion.